### PR TITLE
feat: create proto for tasks

### PR DESCRIPTION
**Pull request by esoubiran-aneo**

### CODE REVIEW

File: Protos/V1/tasks_service.proto [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+  repeated string parent_task_ids = 4;
+  repeated string data_dependencies = 5;
+  repeated string expected_output_ids = 6;
+  repeated string retry_if_ids = 7;
+
+  TaskStatus.TaskStatus status = 8;
+  string status_message = 9;
+
+  TaskOptions.TaskOptions options = 10;
```
Comment (jfonseca-aneo) [inline Protos/V1/tasks_service.proto:38]: After we updated the packages names to lower case this should be referenced as task_status.TaskStatus

File: Protos/V1/tasks_service.proto [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+ */
+message TaskRaw {
+  string id = 1;
+  string session_id = 2;
+  string owner_pod_id = 3;
+
+  repeated string parent_task_ids = 4;
+  repeated string data_dependencies = 5;
+  repeated string expected_output_ids = 6;
+  repeated string retry_if_ids = 7;
```
[the remark this answers is not in the document]
Comment (esoubiran-aneo) [reply Protos/V1/tasks_service.proto:33]: Yes, bien vu

File: Protos/V1/tasks_service.proto [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  TaskOptions.TaskOptions options = 10;
+
+  google.protobuf.Timestamp created_at = 11;
+  google.protobuf.Timestamp submitted_at = 12;
+  google.protobuf.Timestamp started_at = 13;
+  google.protobuf.Timestamp ended_at = 14;
+  google.protobuf.Timestamp pod_ttl = 15;
+
+  message Output {
+    boolean sucess = 1;
```
[the remark this answers is not in the document]
Comment (esoubiran-aneo) [reply Protos/V1/tasks_service.proto:47]: done

File: Protos/V1/tasks_service.proto (hunk 1 above)
Comment (esoubiran-aneo) [reply Protos/V1/tasks_service.proto:38]: Yes, will change this

File: Protos/V1/tasks_service.proto [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  Output output = 16;
+}
+
+/**
+ * A task object
+ */
+message Task {
+  string id = 1;
+  string session_id = 2;
+  TaskOptions.TaskOptions options = 3;
```
[the remark this answers is not in the document]
Comment (esoubiran-aneo) [reply Protos/V1/tasks_service.proto]: changed

**Review by jfonseca-aneo — APPROVED** [review]

**Pull request merged**